packages/shared/lpc/tests/legacy_remap.test.ts

```
@@ -0,0 +1,102 @@
+    for (const slot of legacyCatalog.slots) {
+      for (const assetId of slot.assetIds) {
+        const tagPath = assetId.replace(/\//g, ':');
+        entries.push({ tag: `lpc:${tagPath}:walk`, category: 'lpc', ext: 'webp' });
+      }
+    }
+
+    const result = buildLpcCatalog({ entries });
+
+    const mismatchedSlots = legacyCatalog.slots.filter((legacySlot) => {
+      const derivedSlot = result.slots.find((s) => s.slot === legacySlot.slot);
+      if (!derivedSlot) return false;
```

Biome (2.5.7)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)